ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/v2/package.scala

```diff
@@ -37,12 +38,14 @@ package object v2 {
 
     final case class CommandAccepted(
         offset: LedgerOffset.Absolute,
-        commandId: CommandId,
+        recordTime: Instant,
+        commandId: Option[CommandId],
```

Comment:
Why is the `commandId` optional? Seems like we shouldn't produce/store completions without a command ID, which is useless anyway because you wouldn't be able to connect the completion to a command you sent earlier.

Reply:
right, that was my conclusion as well. I will change it.